manager/integration/tests/test_basic.py

```diff
@@ -3006,6 +3008,163 @@ def test_backup_lock_creation_during_deletion(set_random_backupstore, client, co
+    volume = common.wait_for_volume_healthy(client, volume.name)
+
+    new_data = {
+        'pos': 0,
+        'content': generate_random_data(20 * Mi),
+    }
+
+    _, backup, snapshot, written_data = create_backup(
+        client, volume_name, data=new_data)
+
+    original_ownerID = backup.ownerID
+
+    instance_managers = client.list_instance_manager()
+    engine_instance_manager = None
+    for im in instance_managers:
+        if im.managerType != "engine":
+            continue
+
+        if im.nodeID != backup.ownerID:
+            continue
+
+        engine_instance_manager = im
+    assert engine_instance_manager is not None, \
+        f'expect to find engine instance manager for {backup.name}'
+
+    def finalizer():
+        set_node_cordon(core_api, engine_instance_manager.nodeID, False)
+
+    request.addfinalizer(finalizer)
+
+    set_node_cordon(core_api, engine_instance_manager.nodeID, True)
+
+    delete_and_wait_pod(
+        core_api,
+        engine_instance_manager.name,
+        namespace=common.LONGHORN_NAMESPACE
+    )
+
+    for _ in range(RETRY_COUNTS):
+        engine_instance_manager = client.by_id_instance_manager(
+            engine_instance_manager.name
+        )
+        if engine_instance_manager.currentState == "error":
+            break
+        time.sleep(RETRY_INTERVAL)
+    assert engine_instance_manager.currentState == "error"
+
+    for _ in range(RETRY_COUNTS):
+        _, backup = find_backup(client, volume.name, snapshot.name)
+        if backup.ownerID != original_ownerID:
+            break
+        time.sleep(RETRY_INTERVAL)
+
+    backup_volume, backup = find_backup(client, volume.name, snapshot.name)
+    assert backup.ownerID != original_ownerID, \
+        f'expecting {backup.name} ownerID {original_ownerID} to change'
+
+    wait_for_backup_completion(client, volume.name, snapshot.name)
+    volume.detach(hostId="")
+
+    if recover:
+        set_node_cordon(core_api, engine_instance_manager.nodeID, False)
+
+    return volume, snapshot, backup, backup_volume, written_data
+
+
+def test_backup_delete_when_instance_manager_is_down(set_random_backupstore, client, core_api, volume_name, request):  # NOQA
+    """
+    Scenario: test backup delete when the engine instance manager is down.
+
+    Given backup_ownership_transfer_when_instance_manager_is_down
+
+    When delete backup
+
+    Then back should not exist
+    """
+    volume, snapshot, backup, backup_volume, _ = \
+        backup_ownership_transfer_when_instance_manager_is_down(
+            client, core_api, volume_name, request,
+        )
+
+    common.delete_backup(client, volume.name, backup.name)
+
+    try:
+        _, backup = common.find_backup(client, volume.name, snapshot.name)
+    except AssertionError:
+        backup = None
+    assert backup is None
+
+
+def test_backup_restore_when_instance_manager_is_down(set_random_backupstore, client, core_api, volume_name, request):  # NOQA
```

Comment:
The same reason as the previous comment for `test_backup_delete_when_instance_manager_is_down`.

Restoring a backup to a volume should be related to volume controller instead of backup controller? Should we need to test this here?